### PR TITLE
Review readme for TLS and Downscaling

### PR DESCRIPTION
**Pull request by pmatulis**
This is a review of the TLS and Downscaling sections in the README.

In a future PR, parts of the README will be migrated to the actual documentation (`docs` directory and RST format).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,37 +83,47 @@ microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluste
...
+
+    microovn cluster remove <member_name>
+
+The value of <member_name> is taken from the **Name** column in the output
+of the `microovn cluster list` command.
+
+Any chassis components (`ovn-controller` and `ovs-vswitchd`) running on the
+member will first be stopped and disabled (prevented from starting). For a
+member with central components present (`ovn-southd` and `ovn-northd`), the
+Northbound and Southbound databases will be gracefully removed.
```
Comment (mkalcok) [inline README.md:119]:
```suggestion
member with central components present (`microovn.central`), the
Northbound and Southbound databases will be gracefully removed.
```
In MicroOVN setup, all three processes (`northd`, `ovn-ovsdb-server-sb` and `ovn-ovsdb-server-nb`) are hidden under an umbrella service `microovn.central`

File: README.md [hunk 2]
```diff
@@ -83,37 +83,47 @@ microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluste
...
-that the member was properly removed.
+Any issues that arise during the removal process will need to be resolved
+manually.
 
-```shell
+### Remove a cluster member
+
+To remove a cluster member:
+
+    microovn cluster remove <member_name>
```
Comment (mkalcok) [inline README.md:111]: Is this a preferred way to displaying code blocks?

**Review by mkalcok — CHANGES_REQUESTED** [review]
Just one suggestion otherwise LGTM. Thank you for reviewing this doc

File: README.md (hunk 2 above)
Comment (pmatulis) [reply README.md:111]: For single or a few lines, yes. Many lines would be bound by triple backticks. At this time, the [Charm Guide](https://docs.openstack.org/charm-guide/latest/community/doc-contrib/md.html#code-blocks) is the reference for OpenStack org formatting.

File: README.md (hunk 1 above)
Comment (pmatulis) [reply README.md:119]: I see, interesting. Thank you.

**Review by mkalcok — APPROVED** [review]
LGTM, thanks Peter.

**Pull request merged**